src/lime/extension/Main.hx

```diff
@@ -554,38 +567,33 @@ class Main {
 		
 		if (!hasProjectFile || !isProviderActive) return;
 		
-		var projectFile = getProjectFile ();
-		var buildConfigFlags = getBuildConfigFlags ();
-		var targetFlags = getTargetFlags ();
-		
-		var commandLine = "lime " + getCommandArguments ("display").join (" ");
+		var commandLine = limeCommand + " " + getCommandArguments ("display").join (" ");
 		commandLine = StringTools.replace (commandLine, "-verbose", "");
-		
-		//trace ("Running display command: " + commandLine);
-		
-		try {
-			
-			ChildProcess.exec (commandLine, { cwd: workspace.rootPath }, function (err, stdout:Buffer, stderror) {
-				
-				try {
-					
-					displayArgumentsProvider.update (stdout.toString ());
-					
-				} catch (e:Dynamic) {
-					
-					trace ("Error running display command: " + commandLine);
-					trace (e);
-					
-				}
-				
-			});
-			
-		} catch (e:Dynamic) {
-			
-			trace ("Error running display command: " + commandLine);
-			trace (e);
+
+		ChildProcess.exec (commandLine, { cwd: workspace.workspaceFolders[0].uri.fsPath }, function (err, stdout:Buffer, stderror) {
 			
-		}
+			if (err != null && err.code != 0) {
+	
+				var message = 'Lime completion setup failed. Is the lime command available? Try running "lime setup" or changing the "lime.command" setting.';
+				var showFullErrorLabel = "Show Full Error";
+				window.showErrorMessage (message, showFullErrorLabel).then (function (selection) {
+
+					if (selection == showFullErrorLabel) {
+
+						commands.executeCommand ("workbench.action.toggleDevTools");
+
+					}
+
+				});
+				trace (err);
```

Comment:
We now show the mentioned popup, as well as printing the full error to the dev console.